### ISSUE TITLE
cant set parent circle

### ISSUE THREAD

**Issue by johnabuck** [agilepro/cognoscenti#420]
I tried to set a parent circle for a new class I am running and get this message:
![image](https://user-images.githubusercontent.com/10082365/64832186-d963e200-d5a6-11e9-941a-5255e5df50f4.png)

I get this error message:

![image](https://user-images.githubusercontent.com/10082365/64832269-365f9800-d5a7-11e9-88a5-99c4a13dc010.png)

**Comment by agilepro** [comment]
This was caused by some file system glitch that confused Weaver into thinking there were two workspaces with the same name.  advanced-facilitation-class-fa and advanced-facilitation-class-fa-1.   Accessing one was sometimes accessing the other.  I removed the bogus duplicate workspace and all should be better now.

I will work to try and find the underlying bug that caused the two workspaces with the same name.  i have removed some code that might have caused it, though I don't know exactly what conditions caused it.
